themes/bootstrap3/templates/RecordDriver/DefaultRecord/core.phtml

```diff
@@ -41,6 +41,10 @@
       <?=$this->extraControls['availabilityInfo'] ?? ''?>
     <?php endif; ?>
 
+    <?php if (!empty($this->config()->get('config')->Record->display_versions)): ?>
```

Comment:
I can live with this if the alternative is too complicated to manage, but do we really want display_versions to be a global setting, or would it be better to tie it to the appropriate search options? Otherwise, it may be difficult to do something like turn it on for Search2 but off for Solr.

Reply:
You're right. I need to revisit that.

Reply:
Maybe it's even worth starting by seeing if we can abstract these display_versions calls into a method call everywhere they currently exist, so if we want to change or customize the logic in future, it will be easier to do so.

Reply:
Options class is now used to determine versions availability. display_versions setting is now in searches.ini/Search2.ini.